ts/Extensions/RangeSelector.ts

```diff
@@ -527,6 +535,8 @@ extend(defaultOptions, {
          * The first argument passed is a value to parse,
          * second is a boolean indicating use of the UTC time.
          *
+         * This will only get called for inputs of type `text`.
```

Comment:
Should add info since which version it does happen?

Please, don't forget to add this info in changelog about breaking changes 👍

Reply:
Done